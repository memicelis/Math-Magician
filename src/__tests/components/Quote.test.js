import React from 'react';
import Quote from '../../components/Quote';
import renderer from 'react-test-renderer';

describe('testing Quote component functionality', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
