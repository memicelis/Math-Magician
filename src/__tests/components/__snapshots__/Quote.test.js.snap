// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Quote component functionality renders correctly 1`] = `
<div>
  Loading...
</div>
`;
